__tests__/integration/hathorwallet_facade.test.js

```diff
@@ -1539,6 +1534,7 @@ describe('sendManyOutputsTransaction', () => {
     // Confirm that now the balance is available
     const sendTx = await hWallet.sendTransaction(await hWallet.getAddressAtIndex(4), 8);
     expect(sendTx).toHaveProperty('hash');
+    await waitForTxReceived(hWallet, sendTx.hash);
```

Comment:
We probably don't need to wait for this transaction, as the test is focused on the response object alone. It is not checking the side effects on wallet history.

Reply:
Done https://github.com/HathorNetwork/hathor-wallet-lib/pull/590/commits/6ca745ce49d22bb18ee519203059db62f7700a4a